rct229/rules/section6/section6rule5.py

```diff
@@ -0,0 +1,146 @@
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.ruleset_functions.compare_schedules import compare_schedules
+from rct229.ruleset_functions.get_avg_zone_height import get_avg_zone_height
+from rct229.ruleset_functions.normalize_interior_lighting_schedules import (
+    normalize_interior_lighting_schedules,
+)
+from rct229.schema.config import ureg
+from rct229.utils.jsonpath_utils import find_all
+from rct229.utils.masks import invert_mask
+from rct229.utils.pint_utils import ZERO, pint_sum
+
+BUILDING_AREA_CUTTOFF = ureg("5000 ft2")
+
+
+class Section6Rule5(RuleDefinitionListIndexedBase):
+    """Rule 5 of ASHRAE 90.1-2019 Appendix G Section 6 (Lighting)"""
+
+    def __init__(self):
+        super(Section6Rule5, self).__init__(
+            id="6-5",
+            rmrs_used=UserBaselineProposedVals(False, True, True),
+            each_rule=Section6Rule5.RulesetModelInstanceRule(),
+            index_rmr="baseline",
+            description="Baseline building is modeled with automatic shutoff controls in buildings >5000 sq.ft.",
+            required_fields={
+                "$": ["calendar"],
+                "calendar": ["is_leap_year"],
+            },
+            list_path="ruleset_model_instances[0].buildings[*]",
```

Comment:
`list_path="ruleset_model_instances[*]`

Adding building will point the nested class to building level.
In this case, the search for schedules will be in buildings not in RMI - which causes validation error.